tern/analyze/default/container/single_layer.py

```diff
@@ -42,14 +42,23 @@ def find_os_release(host_path):
     # file exists at this point, try to read it
     with open(etc_path, 'r') as f:
         lines = f.readlines()
-    pretty_name = ''
+        
     # iterate through os-release file to find OS
+    os_release_dict = {}
     for line in lines:
         key, val = line.rstrip().split('=', 1)
-        if key == "PRETTY_NAME":
-            pretty_name = val
-            break
-    return pretty_name.strip('"')
+        os_release_dict[key] = val.strip('"')
+        
+    pretty_name = ''
+    if "PRETTY_NAME" in os_release_dict.keys() :
+        if os_release_dict["PRETTY_NAME"]  == "Distroless" :  
+             #to read version here
+            pretty_name += "{0}-{1}".format(os_release_dict["PRETTY_NAME"], os_release_dict['VERSION'])
```

Comment:
`{0} - {1}` -- space on either side of dash. You will also need to make sure that the line is not longer than 72 characters.